### ISSUE TITLE
Auto save - Enhancement Request

### ISSUE THREAD

**Issue by malcolmjay** [monocasual/giada#380]
An auto-save feature based on a user set time interval would be useful in the event a user comes across an undiscovered bug that crashes the application and results in lost work.

**Comment by tomek-szczesny** [comment]
Good idea! But how exactly is this supposed to work?
I think that this should apply only to already saved projects, and periodically overwrite a *_autosave.gprj/ next to the original.

**Comment by malcolmjay** [comment]
I agree with that approach. It should definitely be conditional based on the file already being saved. Having a master save file and auto save file definitely makes sense. Manual save actions by the user could be written to the master file while the auto save could overwrite the auto save file.

**Comment by monocasual** [comment]
Nice. Let's put this one in the wishlist.
